core/storage/user/gae_models.py

```diff
@@ -2235,9 +2235,9 @@ def get_deletion_policy():
 
     @staticmethod
     def get_export_policy():
-        """Model does not need to exported as it hold authentication details
-        relavant only for backend, and does not contain any exportable
-        user data.
+        """Currently the model hold authentication details relavant only for
+        backend, and no exportable user data, but it may contain future user
```

Comment:
The latter part of this sentence doesn't make sense to me: I'm not sure how tests are relevant to whether this model has user data. Maybe instead: ``.. and no exportable user data. It may contain user data in the future.``

The mention about tests are really a separate discussion not particularly relevant to this model, but instead relevant to the broader problem of auditing changes to models that may require wipeout & takeout changes.

Reply:
Change done. And I understood now what you wanted to say about the tests.